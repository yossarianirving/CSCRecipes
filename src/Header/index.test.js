import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { Header } from '.';

describe('Header', () => {
  const openPage = () => {

  }
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header openPage={openPage}/>, div)
    ReactDOM.unmountComponentAtNode(div);
  })

  test('has a valid snapshot', () => {
    const component = renderer.create(
    <Header openPage={openPage}/>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    });
})