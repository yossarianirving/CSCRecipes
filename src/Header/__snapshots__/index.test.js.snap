// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header has a valid snapshot 1`] = `
<header>
  <h1>
    Recipes
  </h1>
  <nav>
    <a
      href="#"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="#recipes"
      onClick={[Function]}
    >
      All Recipes
    </a>
  </nav>
</header>
`;
